projects/ngx-datatable/src/lib/themes/material.scss

```diff
@@ -51,6 +51,7 @@ $datatable-body-cell-color: rgba(0, 0, 0, 0.87) !default;
 $datatable-group-header-background: #f5f5f5 !default;
 $datatable-group-header-border-top-color: #d9d8d9 !default;
 $datatable-group-header-border-bottom-color: #d9d8d9 !default;
+$datatable-body-row-border-bottom-color: #d9d8d9 !default;
```

Comment:
Just for my understanding: why is this new variable required? Can't it be derived from `$datatable-group-header-border-top-color`/`$datatable-group-header-border-bottom-color`?

Reply:
We introduced the color to be able to see the border in the demos. Just with a white bg, we cannot easily see the sizing of the rows. A new variable is just to provide the option to color it differently to the group borders, if needed - wanted.